addon/addon/models/sprite-tree.ts

```diff
@@ -122,14 +129,21 @@ export default class SpriteTree {
     }
   }
   addSpriteModifier(spriteModifier: SpriteModel): SpriteTreeNode {
-    let parentNode = this.findParentNode(spriteModifier.element);
-    let node = new SpriteTreeNode(
-      spriteModifier,
-      SpriteTreeNodeType.Sprite,
-      parentNode || this
-    );
-    this.nodesByElement.set(spriteModifier.element, node);
-    return node;
+    let existingNode = this.nodesByElement.get(spriteModifier.element);
+
+    if (existingNode) {
+      existingNode.nodeType.add(SpriteTreeNodeType.Sprite);
+      return existingNode;
+    } else {
```

Comment:
https://github.com/cardstack/animations-experiment/pull/45/files#r895357608

is because we don't add the model for the second item we add to a SpriteTreeNode.

Reply:
I suppose we could add both similar to the types, though I'm not sure if there's a clear benefit as it just wraps the element (which is identical).